packages/fx-core/src/plugins/resource/aad/utils/telemetry.ts

```diff
@@ -15,31 +15,40 @@ export class TelemetryUtils {
     eventName: string,
     properties?: { [key: string]: string },
     measurements?: { [key: string]: number }
-  ) {
+  ): void {
     if (!properties) {
       properties = {};
     }
-    properties[Constants.telemetryComponent] = Plugins.pluginNameComplex;
+    properties[Telemetry.component] = Plugins.pluginNameComplex;
+    properties[Telemetry.isSuccess] = Telemetry.success;
     TelemetryUtils.ctx.telemetryReporter?.sendTelemetryEvent(
       eventName,
       properties,
       measurements
     );
   }
 
-  public static sendException(
-    error: Error,
+  public static sendErrorEvent(
+    eventName: string,
+    errorCode: string,
```

Comment:
It's kind of confusing the param name is errorCode but you actually pass error name value here, better rename it.

Reply:
Fixed. Thanks!